packages/cloudflare/src/cli/templates/worker.ts

```diff
@@ -1,46 +1,64 @@
+import { AsyncLocalStorage } from "node:async_hooks";
 import Stream from "node:stream";
 import type { NextConfig } from "next";
 import { NodeNextRequest, NodeNextResponse } from "next/dist/server/base-http/node";
 import { MockedResponse } from "next/dist/server/lib/mock-request";
 import NextNodeServer, { NodeRequestHandler } from "next/dist/server/next-server";
 import type { IncomingMessage } from "node:http";
+import type { CloudflareContext } from "../../api";
 
 const NON_BODY_RESPONSES = new Set([101, 204, 205, 304]);
 
+const cloudflareContextALS = new AsyncLocalStorage<CloudflareContext>();
+
+(globalThis as Record<symbol, unknown>)[Symbol.for("__cloudflare-context__")] = new Proxy(
```

Comment:
nits:
- `as Record<symbol, unknown>` -> `as any` ?
- `Symbol.for("__cloudflare-context__")` should be defined once (in `get-cloudflare-context.ts`) and used at both places. An alternative could be to add a `setRequestContext()`?

Reply:
updated to `any` 👍

Reply:
regarding the symbol, I agree that repeating it is not ideal, but mixing the template files and the api files seems a bit of a messy thing to me that's why I am trying to avoid that 😕 

anyways I did try to import the symbol from `../../api` but that does not work because once we copy those files `../../api` doesn't resolve correctly to the file we want: 
![Screenshot 2024-09-25 at 14 02 34](https://github.com/user-attachments/assets/1417c504-7cdf-4e9f-882f-eef730468821)

we'd have to deal with different file names and extensions to get it to work:
![Screenshot 2024-09-25 at 14 02 46](https://github.com/user-attachments/assets/060ad877-9098-4128-9bca-4d9b2ed20793)

it does feel like a very slippery slope to take... what do you think? 😕

Reply:
Right.
Maybe add a comment that it should be in sync.

Reply:
comment added 🙂 

both here and in the other file